test/test-network/systemd-networkd-tests.py

```diff
@@ -2450,6 +2450,67 @@ def test_ifb(self):
 
         self.wait_online(['ifb99:degraded'])
 
+    def test_rps_cpu_0(self):
+        copy_network_unit('12-dummy.netdev', '25-rps-cpu-0.link')
+        start_networkd()
+
+        self.wait_links('dummy98')
+
+        output = check_output('cat /sys/class/net/dummy98/queues/rx-0/rps_cpus')
+        print(output)
+        self.assertEqual(output, '1')
+
+    @unittest.skipUnless(os.cpu_count() >= 2, reason="CPU count should be >= 2 to pass this test")
+    def test_rps_cpu_1(self):
+        copy_network_unit('12-dummy.netdev', '25-rps-cpu-1.link')
+        start_networkd()
+
+        self.wait_links('dummy98')
+
+        output = check_output('cat /sys/class/net/dummy98/queues/rx-0/rps_cpus')
+        print(output)
+        self.assertEqual(output, '2')
+
+    @unittest.skipUnless(os.cpu_count() >= 2, reason="CPU count should be >= 2 to pass this test")
+    def test_rps_cpu_0_1(self):
+        copy_network_unit('12-dummy.netdev', '25-rps-cpu-0-1.link')
+        start_networkd()
+
+        self.wait_links('dummy98')
+
+        output = check_output('cat /sys/class/net/dummy98/queues/rx-0/rps_cpus')
+        print(output)
+        self.assertEqual(output, '3')
+
+    def test_rps_cpu_all(self):
+        cpu_count = os.cpu_count()
+
+        copy_network_unit('12-dummy.netdev', '25-rps-cpu-all.link')
+        start_networkd()
+
+        self.wait_links('dummy98')
+
+        output = check_output('cat /sys/class/net/dummy98/queues/rx-0/rps_cpus')
+        print(output)
+        if cpu_count == 1:
+            self.assertEqual(output, '1')
+        elif cpu_count == 2:
+            self.assertEqual(output, '3')
+        elif cpu_count == 4:
+            self.assertEqual(output, 'f')
+        elif cpu_count == 8:
+            self.assertEqual(output, 'ff')
```

Comment:
Hm, how about to generate string for arbitrary number of CPUs?
Writing `(1 << cpu_count) - 1` in hex can be used?